tests/middleware/tests.py

```diff
@@ -712,6 +716,58 @@ def get_200_response(req):
         self.assertNotIn("ETag", conditional_get_response)
 
 
+class TestRoutingArgs(SimpleTestCase):
+    def setUp(self):
+        super(TestRoutingArgs, self).setUp()
```

Comment:
explicit TestRoutingArgs, self call inside super might not needed here

Reply:
Yeah true will remove it 👍🏻